### DISCUSSION TITLE
Does need  automatically git commit when executing create-next?

### DISCUSSION THREAD

**Discussion by Rain120** [vercel/next.js#68878] [Ideas]
### Goals

1. `git init` as a prompt
2. not exec `git commit`

### Background

I have two accounts as a git committer, but I need the flexibility to checkout the git user to publish different repos, such as company and personal. so I don't need automatic `git init` by the create tools. Also, I think auto-commit after `git init` is not good for all users who use create. It will make users mistakenly think that their correct account is being used, leading to submission and causing some security risks. Such as the company code or company account for a personal project.

### Proposal

I think we can achieve the goals of the appeal after the discussions

**Comment by Juneezee** [comment]
Hmm...You can use `git reset HEAD~` to undo the commit and commit again using the email you wish to use.

Or, you can have multiple `.gitconfig` and use `includeIf` to control which one is used under different directory, see an example here: https://dzone.com/articles/how-to-use-gitconfigs-includeif
